src/main/java/ca/lambtoncollege/fsdm/s24/auction/repository/AuctionRepository.java

```diff
@@ -54,6 +57,22 @@ public static Auction getAuctionById(int id) throws SQLException {
 
             return fromResultSet(rs);
         }
+    }*/
+
+    public static Auction getAuctionById(int id) throws SQLException {
```

Comment:
did this change anything? if not i think we can restore it to what it was before (remove the comment above)